backend/mobileserver/Makefile

```diff
@@ -4,8 +4,8 @@ include ../../android-env.mk.inc
 # Probably a variation of https://github.com/golang/go/issues/27234 - solution is to build it using vendored deps
 # by turning off GO111MODULE. See also: https://github.com/golang/go/issues/34181#issuecomment-640260162
```

Comment:
Please fix this comment to describe the new situation.

Reply:
Done.